src/containers/AccountConnection.jsx

```diff
@@ -238,6 +238,19 @@ class AccountConnection extends Component {
     }
   }
 
+  forceConnection () {
+    this.setState({submitting: true})
+    this.store.runAccount(this.props.connector, this.state.account)
+    .then(() => this.setState({submitting: false}))
+    .catch((error) => {
```

Comment:
Could we use `.catch(error => this.handleError(error))` here or am I missing a point ?